plugins/pulp_rpm/plugins/migrations/0043_add_modulemd_modulemd-defaults.py

```diff
@@ -0,0 +1,40 @@
+import gzip
+import logging
+
+from pulp.plugins.loader import api as plugin_api
+from pulp.server.controllers import repository as repository_controller
+from pulp.server.db.model import Repository
+from pulp.server.db import connection
+from pulp_rpm.plugins.db.models import YumMetadataFile
+from pulp_rpm.plugins.importers.yum import modularity
```

Comment:
I think we try to write migrations as much independent from the plugin code as possible. It allows to introduce changes to the code without touching migrations.
If you check other migrations most of them don't import anything expect `connection` or `models` if they are needed, the downside is a copy-paste of code.

Thoughts?

Reply:
I agree that in the past we have done that. In this case it was going to be a lot of code to copy/paste. I'll think about to see how we can better mitigate against migrations living longer than the code that they rely on.

Reply:
Good catch, @goosemania.  I think the goal of strict separation is founded on sound principals.  Even though a relatively small amount of code would need to be duplicated, I don't see how it's justified to deviate from that policy in this case.